godtools/XMLContent/KGP/kgp-5.xml

```diff
@@ -100,7 +99,7 @@
                 </content:text>
             </paragraph>
             <paragraph>
-                <content:button type="event" value="kgp-6">
+                <content:button type="event" value="kgp-6,send-information-modal">
```

Comment:
I changed the XML here https://github.com/CruGlobal/godtools-swift/pull/23/files#diff-85a7db7ebbc41f2b16813b30512ea219R88 and here https://github.com/CruGlobal/godtools-swift/pull/23/files#diff-85a7db7ebbc41f2b16813b30512ea219R102